class1.md

```diff
@@ -0,0 +1,261 @@
+
+[click here to go to solutions page](https://github.com/fredhutchio/concepts_machine_learning/blob/master/solutions/class1_solutions.md#write-an-algorithmprotocolruleset-for-making-a-pbj-sandwich)
+
+### When to use machine learning
+
+Just like any other statistical method there are specific situations where applying machine learning is appropriate. Just as important as the algorithm itself is a solid understanding of experimental design, domain expertise in the problem area youre trying to solve, and an understanding of data governance and ethics.
+
+**You cannot code the rules:**
+* Tasks that cannot be solved using a simple, rule-based solution. 
+* When rules depend on too many factors and many of these rules overlap or need to be tuned very finely, it soon becomes difficult for a human to accurately code the rules. You can use ML to effectively solve this problem.
+* This is true for drawing conclusions about highly dimensional data.
+
+**You cannot scale:**
+* You might be able to manually recognize a few hundred emails and decide whether they are spam or not. However, this task becomes tedious for millions of emails. ML solutions are effective at handling large-scale problems.
+
+### Biology is suited for machine learning
+
+<p align="center">
+  <img align="middle" width="600" src="https://imgs.xkcd.com/comics/dna.png">
+</p>
+
+In the past data generation had been a bottleneck in biology research but this is no longer true! Biology research is generating huge amounts of messy and complex data! Now the challenge is figuring out how to integrate datasets and find the underlying patterns to make sense of the massive amounts of data we currently have.
+
+Biology is suited for machine learning applications:
+* You cannot code the rules
+    * Biological systems are incredibly complex with many inputs leading to complex phenotypes. 
+    * Example: if you want to determine whether or not a patient is likely to get cancer we’d need to take many datasets into account (demographic data, health/nutrition data, family history, genetic data, etc). We cannot determine and code the rules for a computer to calculate if a person is likely to get cancer or not.
+
+* You cannot scale
+    * Our human brains cannot make sense of these huge integrated datasets or even just one of them.
+    * Example: a human looking at a series of bases would take years to make sense of the human genome. A computer can find patterns in our genome with relative ease.
+
+### How is machine learning applied?
+
+<img align="right" width="150" src="https://media.giphy.com/media/3oEduS0HyWuqR7kXL2/giphy.gif">
+
+Machine learning applications are all around us! Every time Spotify serves up a customized playlist or radio station based on the music that you like, Facebook loads your newsfeed, your phone autocorrects you, or you interact with a voice assitant like Siri, you are interacting with machine learning algorithms.
+
+Every-day applications:
```

Comment:
I'd include a link here to a source (can be popular media) that talks about ML applied to one or more of these things